app/API/utilities/database/connection.py

```diff
@@ -26,8 +26,8 @@ def initializedb():
         gen_admin = """
                 INSERT INTO
                 users (firstname, lastname, email, password, confirm, admin)
-                VALUES ('main', 'admin', 'admin12@gmail.com', 'passadmin', 'passadmin', true)
-                ON CONFLICT (lastname) DO NOTHING
+                VALUES ('mainadmin', 'admin', 'admin12@gmail.com', 'passadmin', 'passadmin', true)
```

Comment:
Consider not saving `confirm` password in the DB. It is mainly used to check if a user can recall their password